spec/controllers/api/v1/artists_controller_spec.rb

```diff
@@ -36,8 +37,12 @@
   end 
 
   describe "POST#create" do
-    it "creates a new artist" do
+    user = FactoryBot.create(:user)
+    before() do
```

Comment:
we should specify something in the `before`, like `:each` or `:all` depending on what you're trying to achieve
`before(:all)` runs the block one time before all of the examples are run
`before(:each)` runs the block one time before each of your specs in the file

here you can probably say `before(:all)`